packages/discovery-provider/plugins/pedalboard/apps/mri/src/clm.ts

```diff
@@ -17,6 +21,20 @@ export type ClientLabelMetadata = {
     ResourceType: string;
 }
 
+export const ClientLabelMetadataHeader: (keyof ClientLabelMetadata)[] = [
```

Comment:
Added this so we can generate an empty header row for days where there happens to be no data (basically never, even for staging). It is type-safe at least.